AutoCollection/CorrelationContextManager.ts

```diff
@@ -168,7 +168,25 @@ export class CorrelationContextManager {
                 return AppInsightsAsyncCorrelatedErrorWrapper.apply(Object.create(AppInsightsAsyncCorrelatedErrorWrapper.prototype), arguments);
             }
 
+            // Is this object set to rewrite the stack?
+            // If so, we should turn off some Zone stuff that is prone to break
+            var stackRewrite = orig['stackRewrite'];
```

Comment:
I think that might be confusing for this use case. Perhaps I should use dot notation instead and cast to any. The only reason I'm using this bracket notation is to avoid typescript errors on these hidden properties and methods